e2e/pages/CustomFormPage.js

```diff
@@ -0,0 +1,54 @@
+import { By, until } from 'selenium-webdriver';
```

Comment:
shouldn't you rewrite e2e tests to use cypress?

Reply:
Well, the cypress PR was on `master` so in order to do that we should first backport `master` changes into `next`

Reply:
oh, I thought the cypress migration was in `next`. That will be a problem...

Reply:
Why would it be a problem ? We'll have to backport master anyway and tests aren't introducing any changes to the lib